packages/cssnano/src/__tests__/issue927.js

```diff
@@ -13,17 +13,20 @@ p{
 const expected = `div{grid-column:span 2}p{column-count:2}`;
 
 test('it should compress the columns', () => {
-  const processor = postcss([
-    postcss.plugin('cloner', () => {
-      return (css) => {
-        css.walkAtRules((rule) => {
-          css.prepend(rule.clone());
+  const plugin = () => {
```

Comment:
Actually I had left the old style plugins on purpose to check it still worked with those... I guess we can leave one old style and one new style?

Reply:
Ah, I see! I thought they had been forgotten/overseen since they were mostly in issue-specific tests or kind of buried in other things. Perhaps I should have asked and not assumed - sorry about that. Just wanted to save you another issue :-P

Was that the case for all of them or just this one?

Reply:
Let's leave just this one as an old style plugin

Reply:
I left one old style in addition to the new style as you suggested; that way it's clear to anyone viewing it in the future that it's intentional. I also investigated if there is a way to suppress the warning; currently [there is not](https://github.com/postcss/postcss/blob/1545d8aabd2a920c76a70fdd4e7c43a9464fba5e/lib/postcss.js#L31). But I think we can live with that ^^,